core-web/libs/sdk/uve/src/internal/enums.ts

```diff
@@ -0,0 +1,32 @@
+/**
+ * Actions received from the dotCMS UVE
+ *
+ * @export
+ * @enum {string}
+ */
+export enum NOTIFY_CLIENT {
+    /**
+     * Request to page to reload
+     */
+    UVE_RELOAD_PAGE = 'uve-reload-page',
+    /**
+     * Request the bounds for the elements
+     */
+    UVE_REQUEST_BOUNDS = 'uve-request-bounds',
+    /**
+     * Received pong from the editor
+     */
+    UVE_EDITOR_PONG = 'uve-editor-pong',
```

Comment:
We keeping this?

Reply:
Probably not,  I just copy and pasted it

Reply:
Deleted!